### PR TITLE
Feature: add action for overview screen

### PR DESCRIPTION
**Pull request by at-sonvu**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/View/Controllers/PageMenu/PageMenuViewController.swift [hunk 1]
```diff
@@ -34,11 +34,11 @@ class PageMenuViewController: ViewController {
...
         guard (self.navigationController?.navigationBar.frame.size.height) != nil else { return }
         pageMenu = CAPSPageMenu(viewControllers: controllerArray, frame: CGRect(x: 0, y: 64, width: view.frame.width, height: view.frame.height), pageMenuOptions: parameters)
-
         // Lastly add page menu as subview of base view controller view
         // or use pageMenu controller in you view hierachy as desired
-        guard let paegMenu = pageMenu else { return }
-        self.view.addSubview(paegMenu.view)
+        guard let pageMenu = pageMenu else { return }
+        overviewController.pageMenu = pageMenu
+        self.view.addSubview(pageMenu.view)
```
Comment (at-tuanmai2) [inline MyApp/View/Controllers/PageMenu/PageMenuViewController.swift]: remove self

File: MyApp/View/Controllers/PageView/HomeHeaderViewController.swift [hunk 2]
```diff
@@ -34,6 +34,6 @@ class HomeHeaderViewController: ViewController {
         backgroundImageView.clipsToBounds = true
 
         view.addSubview(backgroundImageView)
-        view.sendSubview(toBack: backgroundImageView)
+//        view.sendSubview(toBack: backgroundImageView)
```
Comment (at-tuanmai2) [inline MyApp/View/Controllers/PageView/HomeHeaderViewController.swift]: remove

File: MyApp/View/ViewCustome/CommentFooterSectionView.swift [hunk 3]
```diff
@@ -7,11 +7,12 @@
 //
 
 import UIKit
-
+protocol CommentFooterSectionViewDelegate: class {
+    func moveToPage(_ view: CommentFooterSectionView, index: Int)
```
Comment (at-tuanmai2) [inline MyApp/View/ViewCustome/CommentFooterSectionView.swift]: change func name

File: MyApp/View/ViewCustome/CommentFooterSectionView.swift [hunk 4]
```diff
@@ -7,11 +7,12 @@
...
+}
 class CommentFooterSectionView: UITableViewHeaderFooterView {
-
-    override func awakeFromNib() {
-        super.awakeFromNib()
-        // Initialization code
+    weak var delegate: CommentFooterSectionViewDelegate?
+    @IBAction func loadMoreCommentButtonTouchUpInside(_ sender: UIButton) {
```
Comment (at-tuanmai2) [inline MyApp/View/ViewCustome/CommentFooterSectionView.swift]: add 1 line

File: MyApp/View/ViewCustome/DetailFooterSectionView.swift [hunk 5]
```diff
@@ -7,7 +7,13 @@
 //
 
 import UIKit
-
+protocol DetailFooterSectionViewDelegate: class {
+    func movetoPage(_ view: DetailFooterSectionView, index: Int)
```
Comment (at-tuanmai2) [inline MyApp/View/ViewCustome/DetailFooterSectionView.swift]: change func name

**Pull request merged**